model/manifest.go

```diff
@@ -130,6 +130,12 @@ type Manifest struct {
 	// A description of what your plugin is and does.
 	Description string `json:"description,omitempty" yaml:"description,omitempty"`
 
+	// HomepageURL is an optional link to learn more about the plugin.
+	HomepageURL string `json:"homepage_url,omitempty" yaml:"homepage_url,omitempty"`
+
+	// SupportURL is an optional URL where plugin issue can be reported.
```

Comment:
I'm stumbling across the term "plugin issues". What is a "plugin issue"? Why not just "issues"?

Reply:
```suggestion
	// SupportURL is an optional URL where plugin issues can be reported.
```